src/Modeler.js

```diff
@@ -11,7 +11,9 @@ import MetaShapeToolsModule from './features/meta-tools';
 
 export default class Modeler extends Viewer {
 
-    constructor (options = {}) {
+    constructor (options = { canvas: { } }) {
+        options.canvas.id = 'rnw-modeler';
```

Comment:
hmm

Reply:
Do you have second thoughts about the style or about setting a default id in itself?